packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -129,12 +129,41 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     return super.validateCommand();
   }
 
+  void _validateXcodeBuildSettings() {
+    final Map<String, String>? buildSettings = xcodeBuildResult?.xcodeBuildExecution?.buildSettings;
+    if (buildSettings == null) {
+      return;
+    }
+
+    final Map<String, String?> xcodeProjectSettingsMap = <String, String?>{};
+
+    xcodeProjectSettingsMap['iOS App Version'] = buildSettings['MARKETING_VERSION'];
+    xcodeProjectSettingsMap['iOS Build Number'] = buildSettings['CURRENT_PROJECT_VERSION'];
+    xcodeProjectSettingsMap['App Display Name'] = buildSettings['INFOPLIST_KEY_CFBundleDisplayName'];
+    xcodeProjectSettingsMap['Deployment Target'] = buildSettings['IPHONEOS_DEPLOYMENT_TARGET'];
```

Comment:
"Deployment Target" is an Xcode-ism, we can be more specific.
```suggestion
    xcodeProjectSettingsMap['iOS Minimum Version'] = buildSettings['IPHONEOS_DEPLOYMENT_TARGET'];
```

Reply:
`Deployment Target` is the verbiage used in https://docs.flutter.dev/deployment/ios#review-xcode-project-settings. Can a developer figure out what to do if they see "iOS Minimum Version is missing"?

Reply:
Xcode is all over the place with the labels, but yeah I guess stick with `Deployment Target` or `iOS Deployment Target` since that's what we have in our docs.

<img width="581" alt="Screen Shot 2022-10-17 at 2 19 59 PM" src="https://user-images.githubusercontent.com/682784/196285722-7c418ddf-9c82-46c1-bc54-9b07c842217e.png">
____
<img width="317" alt="Screen Shot 2022-10-17 at 2 20 28 PM" src="https://user-images.githubusercontent.com/682784/196285803-4da076db-4ab9-4f40-8453-bae9e1f3b4ff.png">

____

<img width="667" alt="Screen Shot 2022-10-17 at 2 21 20 PM" src="https://user-images.githubusercontent.com/682784/196285948-529af75c-93bb-425f-887a-77963afb780d.png">